src/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.Directory.NotifyFilter.cs

```diff
@@ -255,5 +253,35 @@ public void FileSystemWatcher_Directory_NotifyFilter_ModifyAndDelete()
                 ExpectEvent(watcher, expected, action, cleanup, new string[] { otherDir, dir.Path });
             }
         }
+
+        [Fact]
+        public void FileSystemWatcher_Directory_NotifyFilter_DirectoryNameDoesntTriggerOnFileEvent()
```

Comment:
Do we have a test for the inverse (a file name not triggering for directory filters)?